sql-cli/sql_cli/__main__.py

```diff
@@ -48,6 +48,32 @@ def about() -> None:
     rprint("Find out more: https://docs.astronomer.io/astro/cli/sql-cli")
 
 
+@app.command(
+    cls=AstroCommand,
+    help="""
+    Gets the value of the configuration key set in the configuration file for the given environment.
+    """,
+)
+def config(
+    project_dir: Path = typer.Option(
+        None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
+    ),
+    env: str = typer.Option(
+        default="default",
+        help="(Optional) Environment used to fetch the configuration key from.",
+    ),
+    key: str = typer.Option(
+        default=None,
+        help="(Optional) Key from the configuration whose value needs to be fetched.",
+    ),
+) -> None:
+    from sql_cli.configuration import Config
+
+    project_dir_absolute = project_dir.resolve() if project_dir else Path.cwd()
```

Comment:
Can you put a comment here to explain why we default to the current working directory?

Reply:
Since this behaviour happens in most - if not all of our commands - perhaps we could separate it into a function.

Reply:
@dimberman If project directory is not given, we assume that the user is calling the command from within the project directory and hence we default to CWD. Moved it to a separate functions, since it is common across most commands and added a comment there on why we use CWD.